.helm/ecamp3/values.yaml

```diff
@@ -30,7 +30,7 @@ php:
   appEnv: prod
   appDebug: "0"
   appSecret: ""
-  corsAllowOrigin: "^https://.*?\\.chart-example\\.local$"
+  corsAllowOrigin:
```

Comment:
Maybe add a comment which still indicates (via an example or written description) that this value may be a regex, may assert on the protocol, etc.